Jenkinsfile

```diff
@@ -1,211 +1,16 @@
 @Library('socrata-pipeline-library')
 
-import com.socrata.ReleaseMetadataService
-def rmsSupportedEnvironment = com.socrata.ReleaseMetadataService.SupportedEnvironment
+Map pipelineParams = [
+  projectName: 'query-coordinator',
+  language: 'scala',
+  defaultBuildWorker: 'build-worker-pg13',
+  numberOfBuildsToKeep: 50,
+  teamsChannelWebhookId: 'WORKFLOW_IQ',
+  projectWorkingDirectory: 'query-coordinator',
+  languageVersion: '2.12',
+  scalaSubprojectName: "queryCoordinator",
+  scalaSrcJar: "query-coordinator/target/query-coordinator-assembly.jar",
```

Comment:
No interpolation is necessary here, so you can change the double quotes to single. In other words, the value associated with `scalaSrcJar` should be of type String, not GString.

Reply:
Done